pkg/cloudprovider/cloudprovider.go

```diff
@@ -150,7 +150,7 @@ func (c *CloudProvider) Name() string {
 	return "aws"
 }
 
-func getCABundle(ctx context.Context, restConfig *rest.Config) (*string, error) {
+func getCABundle(_ context.Context, restConfig *rest.Config) (*string, error) {
```

Comment:
If you aren't using the `context.Context`, just remove it from the function signature. Apologize if I mislead you earlier, I thought you were trying to adhere to some interface.

Reply:
Sounds good!